### DISCUSSION TITLE
Fluent v9: intend to use preview component Table

### DISCUSSION THREAD

**Discussion by chihiroxc** [microsoft/fluentui#25912] [Q&A] [answered]
Hi team,

I'm from Microsoft Synapse team, I'd like to use Table to replace DetailsList in our product. Is there anything we need to do or notice before using it?

Thanks

**Comment by ling1726** [comment, marked as answer]
Hi @chihiroxc, please note that there are two different 'levels' of table components that are planned:
- Table: https://react.fluentui.dev/?path=/docs/preview-components-table--default
- DataGrid: https://react.fluentui.dev/?path=/docs/preview-components-datagrid--default

The documentation should already explain the two in detail.

Another thing to note is that certain column features that the `DetailsList` has such as column resizing or drag and drop and planned for end of Q1 2023

**Reply by JustSlone** [reply]
Also note that preview components which are under the "unstable" import may have breaking changes in future minor releases. If used in a limited situation this should not be an issue but may incur some additional cost as you pickup new versions of the library.

**Reply by chihiroxc** [reply]
Thanks! We'd like to use `Table` mainly because we want to implement the column virtualization, we need to render a big table, like 200 columns. Looks like we should use `Table + react-window` to do that. 

Is there a target release date for Table? (when it's stable and all features that `DetailsList` has are supported)
